core/src/main/java/com/segment/analytics/kotlin/core/platform/Mediator.kt

```diff
@@ -44,6 +47,11 @@ internal class Mediator(internal var plugins: CopyOnWriteArrayList<Plugin> = Cop
                 } catch (t: Throwable) {
                     Analytics.reportInternalError("Caught Exception in plugin: $t")
                     Analytics.segmentLog("Skipping plugin due to Exception: $plugin", kind = LogKind.WARNING)
+                    Telemetry.error("analytics_mobile.integration.invoke.error",
```

Comment:
can we combine all these three logging mechanism to be one call? feels redundant to see all 3 logging for the same error everywhere